src/odemis/gui/cont/acquisition/cryo_acq.py

```diff
@@ -288,6 +292,11 @@ def _acquire_at_features(self, _: wx.Event):
         # the acquisition is started
         self._tab_data.main.is_acquiring.value = True
 
+        # NOTE: acquisition manager expects all streams to be paused before starting.
+        # this cannot be done in the is_acquiring callback as it can be delayed
+        # until after the acquisition starts
+        self._tab.streambar_controller.pauseStreams()
+
```

Comment:
We need the same part in _on_acquire() function, basically wherever is_acquiring is True